zerver/lib/message.py

```diff
@@ -1269,6 +1278,73 @@ def remove_message_id_from_unread_mgs(state: RawUnreadMessagesResult, message_id
     state["mentions"].discard(message_id)
 
 
+def send_unread_msgs(raw_unread_data: RawUnreadMessagesResult) -> Dict[str, Dict[str, Any]]:
+    unread_data = {}
+    for pm in raw_unread_data["pm_dict"]:
```

Comment:
Should the `pm` local variable here just be `message_id`?  It appears to be the integer message ID.

Reply:
Did this change.